src/components/Square.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+
+const Square = ({
+  i,
+  squares,
+  setSquares,
+  remainingAlphabet,
+  setRemainingAlphabet,
+  errorMessage,
+  setErrorMessage,
+  maxNumberConsonants,
+}) => {
+  const vowels = ['A', 'E', 'I', 'O', 'U'];
+  const totalNumberOfConsonants = 27;
```

Comment:
Yeah now that I see you have this again I think a constants file might be the right call so that any file can grab it from one source of truth. vowels can probably go there too

Reply:
Done.